gax/src/test/java/com/google/api/gax/batching/BatcherImplTest.java

```diff
@@ -887,10 +887,15 @@ public void run() {
       }
 
       try {
-        future.get(100, TimeUnit.MILLISECONDS);
+        future.get(3, TimeUnit.SECONDS);
       } catch (TimeoutException e) {
         assertWithMessage("adding elements to batcher should not be blocked").fail();
       }
+
+      // Mockito recommends using verify() as the ONLY recommended way to interact with Argument
+      // captors - otherwise it may incur in unexpected behaviour
+      Mockito.verify(callContext, Mockito.timeout(0)).withOption(key.capture(), value.capture());
```

Comment:
Do we need the `Mockito.timeout(0)` if the time out value is 0?

Reply:
It is not necessary. I removed it in a new commit